scripts/mods/BotImprovements_Combat/BotImprovements_Combat.lua

```diff
@@ -193,7 +156,7 @@ mod:hook(PingTargetExtension, "set_pinged", function (func, self, pinged, flash,
 		end
 	end
 
-	return func(self, pinged, flash, pinger_unit)
+	return func(self, pinged, flash, pinger_unit, show_outline)
```

Comment:
You can use a vararg expression to make these kind of hooks more robust to new arguments being added/removed:

```suggestion
	return func(self, pinged, flash, pinger_unit, ...)
```